app/controllers/api/functions.php

```diff
@@ -911,8 +915,17 @@
             return $response->dynamic($execution, Response::MODEL_EXECUTION);
         }
 
-        /** Collect environment variables */
-        $vars = \array_merge($function->getAttribute('vars', []), [
+        $vars = [];
+
+        $variables = $dbForProject->find('variables', [
```

Comment:
We should be able to get this data from the function object, using find is much more expensive.